scripts/rhoso_adoc_docs_to_text.py

```diff
@@ -107,10 +108,15 @@ def red_hat_docs_path(
             docinfo_content = f.read()
             tree = ET.fromstring(f"<root>{docinfo_content}</root>")
 
-            if get_xml_element_text(tree, "productnumber") != docs_version:
+            productnumber = get_xml_element_text(tree, "productnumber")
+            if Version(productnumber) != Version(docs_version):
```

Comment:
I like this!:)

Reply:
It was required because most of the new docs say "18" instead of "18.0". This makes them equivalent.

Reply:
Oh ok, nice. I thought it was more of a cosmetic change and preventing issues down the road, but now I understand this is extending the documentation right now.